cookiecutter_context.json

```diff
@@ -0,0 +1,60 @@
+{
+    "formal_name": [
+        "First, we need a formal name for your application. ",
+        "This is the name that will be displayed to humans whenever the ",
+        "name of the application is displayed. It can have spaces and ",
+        "punctuation if you like, and any capitalization will be ",
+        "used as you type it. "
+    ],
+    "app_name": [
+        "Next, we need a name that can serve as a machine-readable ",
+        "Python package name for your application. ",
+        "\n",
+        "This name must be PEP508-compliant. This means the name: ",
+        " * may only contain letters, numbers, hyphens and underscores; ",
+        " * can't contain spaces or punctuation; and ",
+        " * can't start with a hyphen or underscore. "
+    ],
+    "project_name": [
+        "Briefcase can manage projects that contain multiple applications, ",
+        "so we need a Project name. ",
+        "If you're only planning to have one application in this project, ",
```

Comment:
Do we need a paragraph spacer blank line here? Or should these two sentences be on the same line?

Reply:
These two sentences do display on the same line in the output when you run `briefcase new` (although if you want we can add a `\n` character to add a line break).